src/Compilers/CSharp/Test/Semantic/Diagnostics/IInvocationExpressionTests.cs

```diff
@@ -0,0 +1,800 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Linq;
+using Microsoft.CodeAnalysis.CSharp.Symbols;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Microsoft.CodeAnalysis.Semantics;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Semantics
+{
+    public class IInvocationExpressionTests : CSharpTestBase
+    {
```

Comment:
+1

---
In reply to: [64130139](https://github.com/dotnet/roslyn/pull/11450#discussion_r64130139) [](ancestors = 64130139)